### ISSUE TITLE
Clan API: Returning wrong player tag.

### ISSUE THREAD

**Issue by Gr8z** [cr-api/cr-api#16]
http://api.cr-api.com/clan/VJQ0GJ0

Returning: 

```
......
{  
   "name":"Raiven",
   "arena":{  
      "imageURL":"/arena/arena8.png",
      "arena":"Arena 8",
      "arenaID":8,
      "name":"Frozen Peak",
      "trophyLimit":2300
   },
   "role":1,
   "expLevel":7,
   "trophies":2336,
   "donations":0,
   "currentRank":46,
   "previousRank":46,
   "clanChestCrowns":33,
   "tag":"8220RQ9Y",
   "roleName":"Member",
   "score":2336
},
......
```
Actual In game tag:

![screenshot_20170919-091726](https://user-images.githubusercontent.com/4633243/30580415-da5d0f98-9d35-11e7-92b2-5666580cc995.jpg)

**Comment by selfish** [comment]
Thank you, investigating.

**Comment by selfish** [comment]
Resolved in v3.0.5
